hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Filter/ReviewOrgUnitChangesFilter.tsx

```diff
@@ -163,19 +191,16 @@ export const ReviewOrgUnitChangesFilter: FunctionComponent<Props> = ({
                 setSelectedVersionId(newValue.toString());
                 handleChange('source_version_id', newValue);
             }
-            filters.groups = [];
+            // Reset the parent_id and groups params
+            delete newParams.parent_id;
+            delete newParams.groups;
+            redirectToReplace(baseUrl, newParams);
```

Comment:
You should still reset the sate of the groups filter to avoid a bug where it switches to error state:
```suggestion
            // Reset the group filter state
            handleChange('groups', []);
            // Reset the parent_id and groups params
            delete newParams.parent_id;
            delete newParams.groups;
            redirectToReplace(baseUrl, newParams);
```